src/sessions/softphone-session-handler.ts

```diff
@@ -88,6 +89,7 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
         this can happen when LA > 1 and persistentConnection is ON
       */
       if (this.hasActiveSession() && !Object.values(this.conversations).find(c => c.session === this.activeSession)) {
+        console.warn('here are all the sessions: ', sessions);
```

Comment:
This looks like a log that can be removed?

Reply:
oops

Reply:
> This looks like a log that can be removed?

oops